sql/sql_class.h

```diff
@@ -4032,6 +4039,7 @@ class THD : public MDL_context_owner,
     m_persist_variables_init = is_init;
   }
   bool is_persist_variables_init() { return m_persist_variables_init; }
+
```

Comment:
unnecessary change

Reply:
I think this was clang-format, but I'll recheck what happens if I remove it.

Reply:
Yes, it was indeed clang-format.